packages/@react-spectrum/s2/intl/en-US.json

```diff
@@ -25,6 +25,7 @@
   "table.sortAscending": "Sort Ascending",
   "table.sortDescending": "Sort Descending",
   "table.resizeColumn": "Resize column",
+  "tabs.selectorLabel": "Tab selector",
```

Comment:
Do we need these additional strings? We didn't have these in v3. Is the label provided by the app enough?

Reply:
linking to discussion about this https://github.com/adobe/react-spectrum/pull/7600#discussion_r1914211263